blueprints/helper/files/app/helpers/__name__.js

```diff
@@ -1,7 +1,7 @@
 import Ember from 'ember';
 
-export function <%= camelizedModuleName %>(input) {
-  return input;
+export function <%= camelizedModuleName %>(params, hash, options, env) {
```

Comment:
@rwjblue I'm a little confused, all the helpers I see in core have 4 params. Can you point me to some documentation on this?